GLAA.Services.Tests/AutoMapper/When_mapping_view_models_to_Licence_entity.cs

```diff
@@ -427,8 +430,8 @@ public void it_should_map_the_organisation_details_view_model_to_the_licence_ent
             AssertAddress(expectedAddress, result.Address);
 
             Assert.AreEqual(input.LegalStatus.LegalStatus, result.LegalStatus);
-            Assert.AreEqual(input.LegalStatus.CompaniesHouseNumber, result.CompaniesHouseNumber);
-            Assert.AreEqual(input.LegalStatus.CompanyRegistrationDate.Date, result.CompanyRegistrationDate);
+            //Assert.AreEqual(input.LegalStatus.CompaniesHouseNumber, result.CompaniesHouseNumber);
```

Comment:
Can this be deleted?

Reply:
Yes, I had a few areas I was unsure of an answer so left them commented out in case we needed them back within 10 mins!